lib/ts/recipe/oauth2client/index.ts

```diff
@@ -14,12 +14,37 @@
  */
 
 import Recipe from "./recipe";
-import { RecipeInterface, APIInterface, APIOptions } from "./types";
+import { RecipeInterface, APIInterface, APIOptions, ProviderConfigWithOIDCInfo, OAuthTokens } from "./types";
 
 export default class Wrapper {
     static init = Recipe.init;
+
+    static async getAuthorisationRedirectURL(input: { redirectURIOnProviderDashboard: string }) {
+        return await Recipe.getInstanceOrThrowError().recipeInterfaceImpl.getAuthorisationRedirectURL(input);
+    }
+
+    static async exchangeAuthCodeForOAuthTokens(input: {
+        providerConfig: ProviderConfigWithOIDCInfo;
+        redirectURIInfo: {
+            redirectURIOnProviderDashboard: string;
+            redirectURIQueryParams: any;
+            pkceCodeVerifier?: string | undefined;
+        };
+    }) {
```

Comment:
Please make the function signature match other index files. (for all other exposed functions in this file as well)
Check for example the `lib/ts/recipe/session/index.ts`

```suggestion
    static async exchangeAuthCodeForOAuthTokens(
        redirectURIInfo: {
            redirectURIOnProviderDashboard: string;
            redirectURIQueryParams: any;
            pkceCodeVerifier?: string | undefined;
        },
        userContext?: Record<string, any>
    ) {
```

Reply:
The userContext passed to this is the wrong type. Please check the linked file on how we handle it in other recipes.